.github/actions/cleanup/config.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+)
+
+type Config struct {
+	DeleteAll  bool
+	Lifetime   int // lifetime in hours
+	PublicKey  string
+	PrivateKey string
+	ManagerUrl string
+}
+
+func NewConfig() (*Config, error) {
+	deleteAllStr := os.Getenv("CLEAN_ALL")
+	if deleteAllStr == "" {
+		return nil, fmt.Errorf("CLEAN_ALL is not set")
+	}
+	var deleteAll bool
+	switch deleteAllStr {
+	case "true":
+		deleteAll = true
+	case "false":
+		deleteAll = false
+	default:
+		return nil, fmt.Errorf("INPUT_CLEAN_ALL must be either true or false")
+	}
+
+	lifetimeStr := os.Getenv("MAX_PROJECT_LIFETIME")
```

Comment:
as in the previous comment, could we have defaults here and follow the same logic suggested before?

Reply:
Done